rholang/examples/linking/v0.2/packages/LinkedList.rho

```diff
@@ -0,0 +1,149 @@
+export LinkedList in {
+  contract @{*LinkedList | "head"}(@list, return) = {
```

Comment:
I'd rather you used something like a list or tuple, because they're ordered, but this isn't a bad idea.